admin/provisioner/static_test.go

```diff
@@ -17,7 +18,15 @@ func Test_staticProvisioner_Provision(t *testing.T) {
 	pg := pgtestcontainer.New(t)
 	defer pg.Terminate(t)
 
-	db, err := database.Open("postgres", pg.DatabaseURL)
+	secret := make([]byte, 32) // 32 bytes for AES-256
+	_, err := rand.Read(secret)
+	require.NoError(t, err)
+
+	encKeyRing := []*database.EncryptionKey{
+		{ID: "test", Secret: secret},
+	}
```

Comment:
Maybe add a `database.NewRandomKeyring` util func? Or use empty keyrings (so encryption is disabled) for tests?

Reply:
Lets finalize on whether moving forward we allow empty keyrings or not - https://github.com/rilldata/rill/pull/5550#discussion_r1738078868.

Reply:
Responded on the other comment

Reply:
removed use of keyring here and `admin_rbac_test` but kept it in `postgres_test` as it test updating prod variables.